build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-turing/build_kokoro.sh

```diff
@@ -30,7 +30,7 @@ docker_setup
 
 docker run "${DOCKER_RUN_ARGS[@]?}" \
   --gpus all \
-  gcr.io/iree-oss/nvidia@sha256:b27c2feb5fc0e7125c5dd933c65c0f7029c157343c91269a17e892e76dd433a3 \
+  gcr.io/iree-oss/nvidia@sha256:cad35c074dab074b6e36e2d5b4f36a2aba863d36013a93d50131db9bef66ec7f \
```

Comment:
Let's enable cuda in these builds and tests that it works now

Reply:
Enabled the test for the x86-turing